ci/test/06_script_b.sh

```diff
@@ -70,6 +70,7 @@ if [ "${RUN_TIDY}" = "true" ]; then
           " src/util/check.cpp"\
           " src/util/error.cpp"\
           " src/util/exception.cpp"\
+          " src/util/fs.cpp"\
```

Comment:
3665c1bdf7b9d47ccab39916c13a8cd9dfbf0eea

There is no `src/util/fs.cpp` file at this commit. It is renamed in the last commit only.